cmd/ro/main.go

```diff
@@ -10,16 +10,24 @@ import (
 	"os"
 	"strings"
 
+	"net"
```

Comment:
The imports here look off. If you delete the import block and run goimports on it, that should fix it.